app/components/dialogs/prioritize-build-modal.js

```diff
@@ -0,0 +1,104 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import eventually from 'travis/utils/eventually';
+import { reads } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import {
+  bindKeyboardShortcuts,
+  unbindKeyboardShortcuts
+} from 'ember-keyboard-shortcuts';
+
+export default Component.extend({
+  keyboardShortcuts: {
+    'esc': 'toggleConfirmationModal'
+  },
+  flashes: service(),
```

Comment:
Please keep the spacing consistent across the entire file. Separate all blocks and properties by a single empty line, so that they were visually independent and easy to distinguish when scanning. The only case when empty line is not necessary is a group of semantically-similar single-line properties.

Reply:
Done.

Reply:
Perfect, thank you!